### PR TITLE
fix(shell): favicons, og image

### PR DESCRIPTION
**Pull request by hadzhehsen**

### CODE REVIEW

File: apps/shell/src/index.html [hunk 1]
```diff
@@ -9,7 +9,9 @@
       name="viewport"
       content="initial-scale=1.0, maximum-scale=1.0, user-scalable=no, width=device-width"
     />
-    <link rel="shortcut icon" href="assets/favicon.png" />
+    <meta name="og:image" content="assets/opengraph-image.png" />
```
Comment (olegshilov) [inline apps/shell/src/index.html]: add og title, add twitter image

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/shell/src/index.html (hunk 1 above)
Comment (hadzhehsen) [reply apps/shell/src/index.html]: Done

File: apps/governance/src/assets/opengraph-image.png
Comment (olegshilov) [inline apps/governance/src/assets/opengraph-image.png:1]: maybe jpg better because has smaller size?

File: apps/shell/src/assets/opengraph-image.png
Comment (olegshilov) [inline apps/shell/src/assets/opengraph-image.png:1]: lets use this image for all apps by absolute url

File: libs/shell/ui-kit/src/utils/format-eth-decimals.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { BigNumber } from '@ethersproject/bignumber';
-import { formatUnits } from 'ethers/lib/utils';
+import { formatUnits } from '@ethersproject/units';
```
Comment (olegshilov) [inline libs/shell/ui-kit/src/utils/format-eth-decimals.ts]: revert

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: apps/governance/src/assets/opengraph-image.png
Comment (hadzhehsen) [reply apps/governance/src/assets/opengraph-image.png:1]: Done

File: libs/shell/ui-kit/src/utils/format-eth-decimals.ts (hunk 2 above)
Comment (hadzhehsen) [reply libs/shell/ui-kit/src/utils/format-eth-decimals.ts]: Done

**Review by olegshilov — APPROVED** [review]

**Pull request merged**